Library/Formula/dependency-check.rb

```diff
@@ -0,0 +1,41 @@
+class DependencyCheck < Formula
+  desc "OWASP Dependency Check"
+  homepage "https://www.owasp.org/index.php/OWASP_Dependency_Check"
+  url "https://dl.bintray.com/jeremy-long/owasp/"\
```

Comment:
We don't apply the style guide to URLs, unless those URLs include a `tag` or `revision`. It's impractical to force every URL onto two lines just to strictly adhere to 80 characters.